spec/fixtures/csv/met.csv

```diff
@@ -1,8 +1,4 @@
 "Object Number","Is Highlight","Is Public Domain","Object ID","Department","Object Name","Title","Culture","Period","Dynasty","Reign","Portfolio","Artist Role","Artist Prefix","Artist Display Name","Artist Display Bio","Artist Suffix","Artist Alpha Sort","Artist Nationality","Artist Begin Date","Artist End Date","Object Date","Object Begin Date","Object End Date","Medium","Dimensions","Credit Line","Geography Type","City","State","County","Country","Region","Subregion","Locale","Locus","Excavation","River","Classification","Rights and Reproduction","Link Resource","Metadata Date","Repository"
-"74.51.4366","False","True",321381,"Ancient Near Eastern Art","Stamp seal","Quasi-pyramidal seal","Cypriot","Late Cypriot III","","","","Demo Role","Demo Prefix","Demo Display Name","Demo Artist Bio","Demo Suffix","","","1900","2000","ca. 12th�11th century B.C.",-1200,-1000,"Steatite, gray brown","Seal Face: 1.19 x 1.68 cm
```

Comment:
Why are these rows removed?

Reply:
The spec was failing on the ID check with multiple lines. It seemed to grab a random one of the 3 that were in there. This may have been related to the wonky spec difference I had. I'll restore the fixture and try it again.

Reply:
Yeah, it's because there are carriage returns in the table cells. That's why I grabbed the first four lines, because that's equivalent to the headers + 1 record.